docker/exporter/export_runner.py

```diff
@@ -58,6 +57,9 @@ def main():
       if ':' in eco:
         continue
       executor.submit(spawn_ecosystem_exporter, args.work_dir, args.bucket, eco)
+  # Upload a ZIP file containing records from all ecosystems.
+  aggregate_all_vulnerabilities(args.work_dir, args.bucket)
+  clean_work_dir(args.work_dir)
```

Comment:
Maybe also call `clean_work_dir` at the beginning of main, in case something doesn't get clean up properly from a different job

Reply:
good point, updated.